providers/register-an-api.md

```diff
@@ -0,0 +1,88 @@
+# How to register an API user for a customer
+
+This document describes how a consumer can gain access to an API exposed on the Logic Marketplace.
+
+## 1. Access request from the customer
+
+In order to get access to Logic, the consumer needs to be able to access the [Logic Console](console). Today there are 2 different ways to achieve this - setting up a federation with Logic Identity and the consumers own identity store (company or municipal AD/AAD) or requesting to have a user created in the Logic Console user store. Fro both, the consumer needs to contact [support](mailto:logic-admin@kmd.dk) with the following information.
+
+- Company name
+- CVR number of the company
+- Contact information:
+  - Full name of the contact person
+  - Email address
+  - Phone number
+  - Company address
+
+## 1. Create a Logic Subscription
+
+When the consumer/partner is setup with access to the console the consumer will now have to setup a Logic subscription through the [Logic Console](console)
+  
+1. Go to [Logic Console](console)
+2. Choose `Subscriptions` from the menu and then click on `Add Subscription`
+3. Enter a `Subscription Name` of the partners name. A `Starter` subscription is fine. Click on `Create`
+4. Record the `Subscription Id`
+
+You will now have a Logic Subscription. If you want to setup multiple subscriptions for better tracking, you jsn just go ahead and create more and you can switch between subscriptions by returning to the `Subscriptions` page and clicking `Activate` on the subscription you wish to manage.
+
+## 1. Arrange for a Partner/Consumer Access and Client Credentials
+
+To allow the consumer/partner to interact with [Logic Console](console) they will need to be registered for access. This can either be:
+
+- By adding their Active Directory as a downstream provider for [Logic Console](console) (a good option for larger organisations)
+- Adding users to the `Logic Console Users` AAD which is managed by the Logic team
+
+If your API uses Logic Identity for authentication each partner might also require a `Client Credential` which will allow them to programmatically call your API.
+
+Go to the [Logic Issues Dashboard](https://dev.azure.com/kmddk/KMDLoGIC/_dashboards/dashboard/02f46530-28bb-4962-bc2a-a12eabfe383e) and use the `New Work Item` widget to create your request.
```

Comment:
Is this OK to publish here?

Reply:
I think so (for now). Might be sub optimal for an external that they can get access. But there is no other good way just now